docs/constructing-ports.md

```diff
@@ -57,9 +55,12 @@ module MathModule{
   )
 }
 ```
-> Notice how MathModule appears again while defining the ports. 
+> Notice how we define ports in MathModule, which is where we defined MathOp as well. 
```

Comment:
Notice how we use the same namespace (`MathModule`) here, as we used in `MathTypes.fpp`.